python/mpt/src/mpt/ethereum_tries.py

```diff
@@ -31,6 +32,8 @@
     "c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
 )
 
+0x56E81F171BCC55A6FF8345E692C0F86E5B48E01B996CADC001622FB5E363B421
```

Comment:
coucou

Reply:
turns out you can just type stuff inside a codebase?